src/components/SearchBy/SearchBy.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import styles from './SearchBy.scss';
+
+const cx = classNames.bind(styles);
+
+class SearchBy extends Component {
+  constructor() {
+    super();
+    this.state = {
+      searchBy: 'title',
```

Comment:
this info is needed inside SearchBar component. So use state inside the Searchbar for saving type and pass callbakc to this component which is called when you click sime type link